torch/nested/_internal/ops.py

```diff
@@ -212,12 +212,13 @@ def lookup_jagged(func, *args, **kwargs) -> Optional[Callable]:
     return None
 
 
-def extract_kwargs(arg):
+def extract_kwargs(arg, **override_kwargs):
```

Comment:
minor nit: instead of changing the signature here and adding complexity to this simple func, I think it'd be easy enough to deal with this inline below:
```python
...
new_offsets = inp._offsets.to(device=new_values.device)
_tensor_symint_registry[new_offsets] = _tensor_symint_registry[inp._offsets]
inp_kwargs = extract_kwargs(inp)
inp_kwargs["offsets"] = new_offsets

return NestedTensor(new_values, **inp_kwargs)
```

wdyt?

Reply:
nice idea! I like it